src/lib.rs

```diff
@@ -571,18 +616,15 @@ pub fn init(options: &InitOptions) -> Result<(ZWaveManager, mpsc::Receiver<ZWave
     let (mut zwave_manager, rx) = ZWaveManager::new(manager);
     try!(zwave_manager.add_watcher());
 
-    let device = match options.device {
-        Some(ref device) => device as &str,
-        _ => try!(get_default_device())
-    };
-
-    try!(
-        fs::File::open(device).map_err(|err| Error::CannotReadDevice(device.to_string(), err))
-    );
-
-    //println!("found device {}", device);
+    let devices = options.devices.clone().unwrap_or(get_default_devices());
+    for device in devices.iter() {
+        try !(
+            fs::File::open(&device).map_err(|err| Error::CannotReadDevice(device.clone(), err))
+        );
```

Comment:
I'm not sure that we should fail if one of the devices is not readable -- but for now that will be good enough.